src/main.rs

```diff
@@ -19,9 +19,14 @@ struct Arguments {
 
 #[tokio::main]
 async fn main() -> Result<()> {
-    tracing_subscriber::fmt::init();
-
     let args = Arguments::parse();
+    let subscriber = tracing_subscriber::FmtSubscriber::builder()
+        .with_env_filter("info,arak=debug")
```

Comment:
I would still get those from the environment like here: https://github.com/Mintbase/minterop-producer/blob/main/src/config.rs#L63-L76